landing/components/common/CTA.tsx

```diff
@@ -0,0 +1,52 @@
+import { Discord, Telegram, XLogo } from "../svg";
+import { landingGradientCircles } from "@shared/utils";
+import { CommonComponents, UIComponents } from "@shared/components";
+
+const socials = [
+  {
+    icon: <XLogo />,
+    link: "https://x.com/yieldi_xyz",
+  },
+  {
+    icon: <Telegram />,
+    link: "https://t.me/yieldi_labs",
+  },
+  {
+    icon: <Discord />,
+    link: "https://discord.com/invite/8aFPCXPwAd",
+  },
+];
+
+const CTA = () => (
+  <div className="relative flex flex-col gap-[88px] font-gt-america-exp px-8 tablet:px-18 text-neutral-900 text-center font-bold pb-[35px] tablet:pb-[115px] overflow-hidden">
```

Comment:
My preference is that we do not hardcode values and instead enhance tw's default size scaling or use a value that's closest to an existing size value.